src/test/java/emissary/util/shell/ExecutrixTest.java

```diff
@@ -23,35 +23,33 @@
 import java.util.stream.IntStream;
 
 import emissary.test.core.UnitTest;
+import org.apache.commons.lang.SystemUtils;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
 public class ExecutrixTest extends UnitTest {
     private Executrix e;
-    private boolean isWindows = System.getProperty("os.name").indexOf("Window") != -1;
+    private final boolean isWindows = SystemUtils.OS_NAME.contains("Windows");
```

Comment:
Conflict with the removal of windows PR

Reply:
Ah, I wasn't aware of that effort. I'll make changes.

Reply:
That effort is an emissary 8 effort due to breaking compatibility. This PR was slated for the 7 line. If anything, the windows support removal PR would need to rebase if something done in this PR conflicted. This effort should not remove windows support.

Reply:
Whoops. My fault @andrewbp. Sorry about that